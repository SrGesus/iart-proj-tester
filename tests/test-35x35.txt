VD	LV	LH	BD	LV	VB	FC	BB	LH	LV	LV	VD	VD	BC	FD	VD	VC	VE	VD	VD	BC	VB	FB	VC	LH	BD	LH	VD	VC	LH	LH	LH	BD	LV	FC
VC	VE	VD	VB	FB	BC	VD	VE	LV	VB	VB	VC	FD	LH	VD	VD	VC	VC	VB	VB	LH	LV	LH	VC	VE	LV	VD	VE	VD	LH	BD	FE	VB	BB	VE
VC	VE	LH	VB	LH	VB	VE	LH	LH	VD	VC	LV	VC	BB	VC	VB	LV	LH	LH	LV	VC	LV	VD	VD	LH	LV	LH	VD	VB	FC	VD	VC	VB	FB	LH
VB	FC	LV	LV	FD	LV	FE	LV	BC	FC	VB	LH	VC	VE	VB	VC	VB	VE	VC	VC	VD	VD	VD	VC	VB	LV	VE	VD	LH	VD	LV	VD	VE	LV	BB
VD	LH	BD	VC	BB	VC	LH	VB	BB	LV	VC	FE	LH	LH	VB	VC	VB	LV	VD	VC	VB	VC	LH	VC	LH	VC	VB	LV	VE	VB	BD	VC	FD	LV	VE
LV	VD	VE	FB	LH	VE	VE	LV	LV	FD	LV	BD	LV	LH	VB	BC	VD	FC	VE	LV	VD	LV	VC	VE	FC	VE	VB	VB	LV	VD	FB	LH	VB	LV	VB
LV	LV	VC	VC	VB	VE	LV	LH	LV	VC	LH	VD	VB	VD	LH	FD	VE	VB	VE	VE	VE	VB	VE	LH	BC	VE	VD	BC	VC	VD	VE	BE	VE	FD	LV
LV	BB	VD	VB	VE	VB	VC	LV	LV	VE	VC	VD	VD	FE	VE	LH	VD	VB	VB	BE	LH	LV	VB	VB	BC	FC	LV	FC	VC	VE	LV	LV	VB	BC	LV
LV	FC	VC	VB	VC	BC	VD	VC	VC	VE	LV	VC	LV	VC	VE	LV	VB	VE	VB	LV	FB	LH	VE	LH	VD	VE	LH	VD	LH	VD	LH	BD	VB	FE	LV
BB	LH	VE	VC	VE	FC	LH	VB	VB	LH	VE	LV	VC	BE	VE	VE	LH	VC	FE	LV	VE	VC	LH	VB	LH	VC	VD	VB	VD	LH	VE	FE	VD	LH	BB
VD	LV	VD	LV	VD	LH	VE	LH	LH	VD	LH	VD	LV	VB	LV	VC	FE	VC	BE	VD	LV	VE	BD	FE	VB	BD	FC	VD	VC	FD	VB	BC	VE	FD	LV
VE	VD	LH	VE	VC	VC	LV	VC	BB	FB	VC	VD	BB	LH	LH	LH	VD	VB	BB	FE	BE	VC	VC	VD	VE	LV	VE	VB	VB	LV	LH	VC	VC	BC	VE
LV	VB	BD	VC	LH	VB	LV	VD	VC	VC	LH	FD	LH	VE	LH	VD	VE	VC	VD	LV	VC	LV	VB	VB	VC	VE	VD	LV	VE	VE	BC	FB	LV	LH	FE
LH	VD	FE	FC	LV	VD	LV	VE	FC	VE	VE	BD	BD	VC	VE	VD	VE	VD	LV	VB	LH	VB	VD	VB	VC	VB	LH	VB	VB	BD	LH	VC	LV	VD	VB
FD	LV	VB	BB	VE	LV	VB	FB	BD	VB	FD	VB	BB	LH	VC	FC	BB	FE	VD	VB	VD	LH	LV	LH	LV	VD	VD	VB	LH	LH	FB	BC	VD	LH	VB
VB	VE	LV	LV	VB	VB	VC	LV	BE	VC	LH	VC	FB	VD	LH	LV	LV	LV	VB	VE	LH	BC	LV	FD	VE	VE	VE	VC	LV	VB	VB	VB	LV	LV	BB
LV	FC	BB	LH	BB	LV	LH	VC	BC	VD	FC	VC	VD	BE	LV	LV	VC	VB	LV	VB	FC	VE	LV	VC	LV	FB	BB	VB	LH	FD	VB	LV	VB	VC	VC
VB	VB	LH	LH	FC	VC	VB	LH	LH	LH	BC	LV	VC	VE	FD	VB	VE	VB	LV	LV	BC	VE	VD	BD	VE	VB	BC	LH	VD	BB	VC	VE	VD	VC	VC
FD	LH	FC	LH	VE	VB	LH	LH	FD	LV	VC	LV	LV	LH	VD	VE	VB	FE	LV	LH	BE	VC	LH	LV	VE	VE	VB	LV	VD	LH	FE	VD	LV	VD	LH
LV	VE	VB	LH	LH	VC	VE	VD	LV	VE	FE	BD	LV	VB	LH	VE	VD	VD	LV	VB	FC	VC	VE	LH	VB	LV	LV	VD	VE	VB	VD	LV	VC	LH	FC
LH	VD	VD	VE	VE	VE	LH	VD	VD	LH	LH	VC	FE	LV	LH	LV	VB	VB	VB	VE	VD	VD	FE	VE	VD	VB	VE	VB	LH	LH	VE	FB	LV	VB	VB
LV	VE	LV	LV	LH	LV	LH	VC	BC	LV	LV	VE	VD	VB	LH	VE	BE	VC	VE	VC	BE	VB	LH	VB	VC	LV	VE	LH	BB	FE	VB	BC	VE	VE	BD
BE	LH	LV	FD	VC	LH	LH	VB	LH	FD	VB	VD	LH	VD	VC	VB	LV	LH	FC	BC	VC	VD	VD	BD	LV	VD	VC	LH	VE	VE	VB	VB	LH	BC	FD
VD	VE	VB	VB	VE	VE	VB	VC	VE	VB	LV	VC	VD	BC	FB	LV	LV	VB	VB	LH	VD	LV	VC	VE	VE	FC	LH	VC	LV	VE	FD	LV	FE	VE	VB
VB	VC	LH	LV	VC	VD	VD	LH	VB	VD	BB	LH	VB	BD	VC	VD	LH	VC	VD	LV	VD	VD	LH	LH	LH	BD	VC	LH	FB	VD	BB	VE	LV	VE	VC
LV	VE	VE	VB	VE	VE	LV	VE	VC	LV	VE	VC	VE	FC	LV	FB	LV	VC	VD	VB	VD	FB	BD	VE	LH	VB	FC	VE	VD	VE	BE	LH	BC	VB	VE
LH	VC	LV	LV	VE	FD	VC	BC	FE	VC	LV	VB	VB	VE	LV	BE	VB	FC	VE	LV	LH	VE	LV	FC	LV	VC	LV	VE	VB	VD	VD	VC	VD	VD	LH
VB	BB	FE	VB	VD	VC	VE	VB	LH	BD	FC	BD	LH	VB	LH	VB	VB	VD	LH	VC	VB	VD	VD	BE	LH	LV	FE	LV	VE	VE	LH	VC	LH	VB	LV
VB	VC	VC	VB	VD	BD	VC	LH	FC	VB	VC	LV	LH	VE	VC	VD	VD	LH	LH	BB	LV	VE	FD	LV	VC	VD	LH	VC	VC	VD	VB	VB	LV	LH	BB
BB	VB	LH	VC	VE	VC	BD	LV	VE	LH	VE	VC	VE	VD	LV	VC	FB	VB	LH	VD	LV	VD	VE	VE	VC	FE	BC	BE	VD	VD	VB	LV	FC	VB	VE
LV	VE	VC	LH	VD	VC	LH	LV	VE	LH	VB	LH	VB	FD	VD	LV	BC	LV	FC	VD	VC	VE	BD	VC	VE	VB	VE	VE	VB	VD	FD	LH	BB	VC	FE
VD	LH	VB	LH	VB	VB	FC	VD	LH	LH	VD	VB	LH	LV	BC	VE	VD	BD	VB	VD	VC	LV	VE	BD	FD	VD	VD	VC	FE	LV	VB	VD	VE	VB	LH
FE	VB	VE	VD	VC	VD	LH	LV	BC	BC	VD	VC	LV	LH	VE	BC	LV	LH	BE	FD	LH	LH	FD	VD	VE	VD	LH	BB	LV	VC	LH	VE	FD	VB	BB
LV	VC	LV	VD	VE	FB	VB	VE	FB	LV	LV	LV	VB	LV	VC	FC	VD	VC	FD	VC	VD	BD	VC	LH	VC	VC	VE	VE	FD	LH	BE	BB	LV	VE	LH
VC	LH	LH	BB	LV	LV	VB	VE	LH	VB	VC	VC	VC	LV	LH	LV	VC	VC	LH	VC	FE	VB	VB	BE	LV	LV	VD	VD	LH	LH	VD	VD	FB	VE	VD
