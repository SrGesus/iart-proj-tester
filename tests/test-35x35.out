VB	LH	LH	BB	LH	VE	FD	BB	LH	LH	LH	VE	VB	BB	FE	VB	VE	VB	VE	VB	BB	VE	FB	VB	LH	BB	LH	VE	VB	LH	LH	LH	BB	LH	FE
VD	VE	VB	VC	FD	BC	VE	VD	LH	VE	VB	VC	FC	LV	VB	VC	VD	VC	VD	VC	LV	LV	LV	VD	VE	LV	VB	VC	VD	LH	BB	FE	VD	BB	VE
VB	VC	LV	VB	LH	VE	VD	LH	LH	VC	VD	LH	VE	BD	VC	VB	LH	LH	LH	LH	VC	LV	VD	VE	LV	LV	LV	VB	VE	FD	VC	VB	VE	FC	LV
VD	FE	LV	LV	FB	LV	FD	LH	BB	FE	VB	LH	VC	VD	VE	VD	VE	VB	VE	VB	VE	VD	VE	VD	VC	LV	VD	VC	LV	VB	LH	VC	VD	LH	BE
VB	LH	BE	VD	BE	VD	LH	VE	BD	LH	VC	FD	LH	LH	VC	VB	VC	LV	VD	VC	VD	VE	LV	VB	LH	VC	VB	LH	VC	VD	BB	VE	FD	LH	VC
LV	VB	VC	FB	LV	VB	VE	LV	LV	FD	LH	BB	LH	LH	VE	BD	VE	FC	VB	LH	VE	LV	VD	VC	FB	VB	VC	VB	LH	VE	FC	LV	VB	LH	VE
LV	LV	VB	VC	VD	VC	LV	LV	LV	VB	LH	VC	VB	VE	LV	FC	VD	VE	VD	VE	VD	VC	VB	LH	BE	VD	VE	BD	VE	VD	VE	BD	VC	FB	LV
LV	BD	VC	VB	VE	VB	VC	LV	LV	VD	VE	VB	VC	FC	VD	LH	VE	VD	VE	BD	LH	LH	VC	VB	BE	FB	LV	FC	VD	VE	LV	LV	VB	BE	LV
LV	FC	VB	VC	VD	BE	VB	VC	VD	VE	LV	VD	LH	VE	VB	LH	VC	VB	VC	LV	FD	LH	VE	LV	VD	VC	LV	VB	LH	VC	LV	BD	VC	FC	LV
BD	LH	VC	VB	VE	FC	LV	VB	VE	LV	VD	LH	VE	BD	VC	VB	LH	VC	FB	LV	VB	VE	LV	VD	LH	VE	VD	VC	VB	LH	VC	FC	VB	LH	BE
VD	LH	VE	LV	VD	LH	VC	LV	LV	VD	LH	VE	LV	VD	LH	VC	FD	VE	BD	VC	LV	VD	BE	FB	VB	BC	FE	VB	VC	FB	VB	BB	VC	FB	LV
VB	VE	LV	VD	VE	VB	LH	VC	BD	FE	VB	VC	BD	LH	LH	LH	VE	VD	BE	FD	BC	VE	VD	VC	VD	LH	VE	VD	VE	LV	LV	VD	VE	BD	VC
LV	VD	BC	VE	LV	VD	LH	VE	VD	VE	LV	FB	LV	VB	LH	VE	VD	VE	VD	LH	VE	LV	VB	VE	VB	VE	VD	LH	VC	VD	BC	FE	LV	LV	FB
LV	VB	FE	FC	LV	VB	LH	VC	FB	VD	VC	BD	BE	VD	VE	VD	VE	VD	LH	VE	LV	VD	VC	VD	VC	VD	LH	VE	VB	BB	LH	VE	LV	VD	VC
FC	LV	VB	BB	VC	LV	VB	FE	BD	VE	FD	VC	BD	LH	VC	FD	BC	FE	VB	VC	VD	LH	LH	LH	LH	VE	VB	VC	LV	LV	FD	BE	VD	LH	VE
VB	VC	LV	LV	VB	VC	VD	LH	BE	VD	LH	VE	FC	VB	LH	LH	LH	LH	VC	VB	LH	BB	LH	FE	VB	VC	VD	VE	LV	VD	VE	VD	LH	LH	BE
LV	FD	BE	LV	BD	LH	LH	VE	BD	VE	FB	VD	VE	BD	LH	LH	VE	VB	LH	VC	FB	VD	LH	VE	LV	FD	BB	VC	LV	FB	VD	LH	VE	VB	VC
VD	VE	LV	LV	FC	VB	VE	LV	LV	LV	BD	LH	VC	VD	FE	VB	VC	VD	LH	LH	BC	VE	VB	BE	VD	VE	BD	LH	VC	BD	VE	VB	VC	VD	VE
FB	LV	FC	LV	VB	VC	LV	LV	FC	LV	VD	LH	LH	LH	VE	VD	VE	FD	LH	LH	BB	VC	LV	LV	VB	VC	VD	LH	VE	LV	FC	VD	LH	VE	LV
LV	VD	VE	LV	LV	VB	VC	VD	LH	VC	FD	BB	LH	VE	LV	VB	VC	VB	LH	VE	FC	VB	VC	LV	VD	LH	LH	VE	VD	VC	VB	LH	VE	LV	FC
LV	VB	VC	VD	VC	VD	LH	VE	VB	LH	LH	VC	FB	LV	LV	LV	VB	VC	VB	VC	VB	VC	FB	VD	VE	VB	VE	VD	LH	LH	VC	FB	LV	VD	VE
LV	VD	LH	LH	LH	LH	LH	VC	BD	LH	LH	VE	VD	VC	LV	VD	BE	VB	VC	VB	BC	VE	LV	VB	VC	LV	VD	LH	BB	FE	VB	BC	VC	VB	BE
BD	LH	LH	FE	VB	LH	LH	VE	LV	FD	VE	VD	LH	VE	VD	VE	LV	LV	FD	BE	VB	VC	VD	BC	LH	VC	VB	LH	VC	VB	VC	VB	LH	BE	FC
VD	VE	VB	VE	VD	VE	VB	VC	VD	VE	LV	VB	VE	BD	FE	LV	LV	VD	VE	LV	VD	LH	VE	VB	VE	FB	LV	VB	LH	VC	FB	LV	FB	VD	VE
VB	VC	LV	LV	VB	VC	VD	LH	VE	VD	BE	LV	VD	BE	VB	VC	LV	VB	VC	LV	VB	VE	LV	LV	LV	BD	VC	LV	FD	VE	BD	VC	LV	VB	VC
LV	VB	VC	VD	VC	VB	LH	VE	VD	LH	VC	VD	VE	FC	LV	FB	LV	VD	VE	VD	VC	FC	BD	VC	LV	VD	FE	VD	VE	VD	BC	LH	BE	VD	VE
LV	VD	LH	LH	VE	FC	VB	BC	FE	VB	LH	VE	VD	VE	LV	BD	VC	FB	VD	LH	LH	VE	LV	FB	LV	VB	LH	VE	VD	VE	VB	VE	VD	VE	LV
VD	BB	FE	VB	VC	VB	VC	VB	LH	BE	FD	BC	LH	VC	LV	VD	VE	VD	LH	VE	VB	VC	VD	BE	LV	LV	FB	LV	VB	VC	LV	VD	LH	VC	LV
VB	VC	VB	VC	VB	BC	VE	LV	FD	VC	VB	LH	LH	VE	VD	VE	VD	LH	LH	BE	LV	VB	FE	LV	VD	VC	LV	VD	VC	VB	VC	VB	LH	LH	BE
BD	VE	LV	VB	VC	VB	BE	LV	VB	LH	VC	VB	VE	VD	LH	VC	FB	VB	LH	VC	LV	VD	VE	VD	VE	FD	BC	BB	VE	VD	VE	LV	FB	VB	VC
LV	VD	VC	LV	VB	VC	LV	LV	VD	LH	VE	LV	VD	FE	VB	LH	BE	LV	FB	VB	VC	VB	BC	VE	VD	VE	VB	VC	VD	VE	FC	LV	BD	VC	FB
VD	LH	VE	LV	VD	VE	FC	VD	LH	LH	VC	VD	LH	LH	BC	VE	VD	BC	VC	VD	VE	LV	VB	BC	FE	VD	VC	VB	FE	LV	VB	VC	VD	VE	LV
FB	VB	VC	VD	VE	VD	LH	LH	BB	BB	VE	VB	LH	LH	VE	BD	LH	LH	BB	FE	LV	LV	FC	VB	VE	VB	LH	BC	LH	VC	LV	VB	FE	VD	BE
LV	VD	LH	VE	VD	FE	VB	VE	FC	LV	LV	LV	VB	LH	VC	FC	VB	VE	FC	VB	VC	BD	VE	LV	VD	VC	VB	VE	FD	LH	BE	BD	LH	VE	LV
VD	LH	LH	BC	LH	LH	VC	VD	LH	VC	VD	VC	VD	LH	LH	LH	VC	VD	LH	VC	FD	VC	VD	BC	LH	LH	VC	VD	LH	LH	VC	VD	FE	VD	VC
